docs/.vuepress/components/LayoutArticle.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
This custom **LayoutArticle.vue** file should be moved to _docs/.vuepress/theme/_, which is the out-of-the-box folder Vuepress provides to keep track of all layouts. I don't want to confuse Vue components we create for layout with components we create to be used inside those layouts, which is what _docs/.vuepress/components/_ is for.